### DISCUSSION TITLE
Abbreviations

### DISCUSSION THREAD

**Discussion by thientrinhcoder** [QilinCloud/QilinCloud#1] [General]
Should have some Abbreviation of Cloud terminologies: 
PaaS, SaaS, IaaS...

**Comment by marc-costea** [comment]
Good idea.
When you creating the [architecture page](https://github.com/QilinCloud/QilinCloud/wiki/Architecture), for each abbrevation that is used there, please also create some explenational entry in the [abbrevations page](https://github.com/QilinCloud/QilinCloud/wiki/Abbreviations). But for sure, it also makes sense to explain other abbrevations that are related to the topic.
